src/vos/vos_container.c

```diff
@@ -934,6 +934,13 @@ vos_cont_get_globla_stable_epoch(daos_handle_t coh)
 	cont = vos_hdl2cont(coh);
 	D_ASSERT(cont != NULL);
 
+	if (cont->vc_pool->vp_pool_df->pd_version < VOS_POOL_DF_2_8) {
+		D_DEBUG(DB_MD, DF_CONT" return 0 stable epoch for lower pool version %d\n",
+			DP_CONT(cont->vc_pool->vp_pool_df->pd_id, cont->vc_id),
+			cont->vc_pool->vp_pool_df->pd_version);
+		return 0;
+	}
```

Comment:
That seems unnecessary since for old pool either `cont_ext` is NULL or `cont_ext->ced_global_stable_epoch` is not set by default.

Reply:
that need some knowledge for example suing umem_zalloc() to allocate the vos_cont_ext_df. adding such explicit code looks harmless?

Reply:
It is harmless but redundant, not fatal.